controls/RHEL-07-040190.rb

```diff
@@ -31,8 +31,8 @@
 If “ClientAliveInterval” is not set to “600” in /etc/ ssh/sshd_config, and a lower value is not documented with the Information System Security Officer (ISSO) as an operational requirement, this is a finding.'
 
 # START_DESCRIBE RHEL-07-040190
-  describe file('') do
-    it { should match // }
+  describe sshd_config do
+    its('ClientAliveInterval') { should eq '600' }
```

Comment:
Yeah this one is hard to say, the STIG recommends a value of 600. It seems even the [openSCAP STIG](https://github.com/OpenSCAP/scap-security-guide/blob/72e79309039d0290b553222afd846a27e32187bf/RHEL/7/input/profiles/stig-rhel7-server-upstream.xml#L22) would raise the same failure if set to anything other than 10 minutes. I think you might need to exempt this control from your tests with approval from your security team.